### DISCUSSION TITLE
Hovering over links not working

### DISCUSSION THREAD

**Discussion by TheAussieTinker** [RafaelGB/obsidian-db-folder#149] [Q&A]
**Describe the bug**
Hovering over a link doesn't bring up the preview of the linked note. You need to hold down cmd/ctrl for it to appear.

**To Reproduce**
1. make a database folder
2. add a link
3. hover over the link
4. nothing shows up until you hold down cmd/ctrl

**Expected behavior**
upon hovering over a link, a preview to the linked note would appear

**Desktop (please complete the following information):**
 - OS: macOS

**Comment by RafaelGB** [comment]
the db view does not support obsidian native events. The ctrl hover is a decision to not break in case of bug the app
